### PR TITLE
MINT-1926 Implement request to /signin/signature/chronomint on profile backend.

### PR DESCRIPTION
**Pull request by wAISw**
Changed profile-backend request.

### CODE REVIEW

File: packages/core/redux/session/selectors/session.js [hunk 1]
```diff
@@ -90,5 +94,42 @@ export const getAccountProfileSummary = createSelector(
...
+    getLitecoinSigner,
+    getNemSigner,
+    getWavesSigner,
+  ],
+  (persistAccount, ethereumSigner, bitcoinSigner, bitcoinCashSigner, litecoinSigner, nemSigner, wavesSigner) => {
+    const { selectedWallet } = persistAccount
+    const signerPath = selectedWallet.encrypted[0].path
+    return [
+      {
+        type: "ethereum-public-key",
```
Comment (ozalexo) [inline packages/core/redux/session/selectors/session.js]: MINOR
Please replace " to single-quotes
The same at line 127

File: packages/core/redux/session/selectors/session.js [hunk 2]
```diff
@@ -90,5 +94,42 @@ export const getAccountProfileSummary = createSelector(
...
+    getEthereumSigner,
+    getBitcoinSigner,
+    getBitcoinCashSigner,
+    getLitecoinSigner,
+    getNemSigner,
+    getWavesSigner,
+  ],
+  (persistAccount, ethereumSigner, bitcoinSigner, bitcoinCashSigner, litecoinSigner, nemSigner, wavesSigner) => {
+    const { selectedWallet } = persistAccount
+    const signerPath = selectedWallet.encrypted[0].path
```
Comment (ozalexo) [inline packages/core/redux/session/selectors/session.js]: Is there a selector for the signer path in persistAccount/selectors? We do not need whole persistAccount here.

File: packages/core/redux/session/selectors/session.js [hunk 3]
```diff
@@ -90,5 +94,42 @@ export const getAccountProfileSummary = createSelector(
...
+    return [
+      {
+        type: "ethereum-public-key",
+        value: ethereumSigner.getAddress(signerPath),
```
Comment (e-maximus) [inline packages/core/redux/session/selectors/session.js]: getAddress can be async

File: packages/core/redux/profile/service.js [hunk 4]
```diff
@@ -4,23 +4,29 @@
...
 const URL_PROFILE_IMAGE_DOWNLOAD = `media/image/`
 const URL_PROFILE_IMAGE_UPLOAD = 'media/image/upload'
 
-const EXCHANGE_PURPOSE_DATA = { purpose: 'exchange' }
+const EXCHANGE_PURPOSE_DATA = { purpose: 'middleware' }
 
 export default class ProfileService {
-  static service = axios.create({ baseURL: URL_PROFILE_BASE_PATH })
+  static init = (networkID) => {
+    const baseURL = `${networkID === NETWORK_MAIN_ID ? URL_PROFILE_HOST : URL_PROFILE_HOST_TESTNET}${URL_PROFILE_BASE_PATH}`
```
Comment (e-maximus) [inline packages/core/redux/profile/service.js]: I'd make a separate variable for ```${networkID === NETWORK_MAIN_ID ? URL_PROFILE_HOST : URL_PROFILE_HOST_TESTNET}``` condition for better understanding

File: packages/login-ui/redux/thunks.js [hunk 5]
```diff
@@ -127,6 +125,8 @@ export const onSubmitLoginForm = (password) => async (dispatch, getState) => {
         const defaultURL = await dispatch(SessionThunks.login(selectedAccount))
 
         dispatch(replace(localStorage.getLastURL() || defaultURL))
+        const signer = getEthereumSigner(getState())
```
Comment (alexander-elgin) [inline packages/login-ui/redux/thunks.js]: There is a lot of duplication in `WALLET_TYPE_MEMORY` and the alternative branches. Many lines of code are like two peas in a pod. It s required to move the duplicated code to reusable functions.

File: packages/core/redux/session/thunks.js [hunk 6]
```diff
@@ -133,7 +134,7 @@ export const login = (account) => async (dispatch, getState) => {
   await dispatch(watcher({ web3 }))
 
   const userManagerDAO = daoByType('UserManager')(getState())
-  const [isCBE, profile /*memberId*/] = await Promise.all([
+  const [isCBE, profile /*memberId*/ ] = await Promise.all([
```
Comment (alexander-elgin) [inline packages/core/redux/session/thunks.js]: remove the comment

File: packages/login-ui/redux/thunks.js (hunk 5 above)
Comment (wAISw) [reply packages/login-ui/redux/thunks.js]: I created a bug task for it (MINT-2087)

**Pull request merged**